nixos/modules/services/misc/gitweb.nix

```diff
@@ -44,6 +52,18 @@ in
       internal = true;
     };
 
+    package = mkOption {
+      default = pkgs.gitweb;
+      type = types.path;
+      internal = true;
+    };
+
+  };
+
+  config = mkIf cfg.gitwebTheme {
+    services.gitweb.package = pkgs.gitweb.override {
+      gitwebTheme = true;
+    };
```

Comment:
This will not allow you to override the package if you're using `gitWebTheme`. Please just override `cfg.package` in the implementation., e.g. `let package = cfg.package.override { ... };`